data/kitsData.js

```diff
@@ -692,6 +693,32 @@ export const kitsData = {
         latestVersion: '0.1.0',
         new: true
       }
+    },
+    {
+      id: 'geometry',
+      name: 'GEOMETRY KIT',
+      logo: GeometryKitLogo,
+      logoHeight: 100,
+      logoWidth: 100,
+      route: '/docs-kits/kits/geometry-kit/adoption-view',
```

Comment:
```suggestion
      route: '/docs-kits/next/kits/geometry-kit/adoption-view',
```
Otherwise the build will fail. The KIT currently only exists within `next`.